test/e2e/app-dir/prefetching-not-found/prefetching-not-found.test.ts

```diff
@@ -24,8 +24,10 @@ describe('prefetching-not-found', () => {
       expect(await browser.elementByCss('h1').text()).toBe('Home Page')
     })
 
-    // assert the same behavior, but starting at the not found page. This is to ensure that when we seed the prefetch cache,
-    // we don't have any cache collisions that would cause the not-found page to remain rendered when following a link to the home page
+    // Assert the same behavior, but starting at the not found page. This is to
+    // ensure that when we seed the prefetch cache, we don't have any cache
+    // collisions that would cause the not-found page to remain rendered when
+    // following a link to the home page.
```

Comment:
This formatting change was done to trigger a deployment test.

Reply:
Th [deployment](https://github.com/vercel/next.js/actions/runs/12787158115/job/35646320825) succeeded.